### DISCUSSION TITLE
[AJUSTE TRILHA] GitHub

### DISCUSSION THREAD

**Discussion by endersonmenezes** [codaqui/institucional#117] [Trilhas de Ensino]
Pessoal, a @thaifurforo está liderando um projeto para a gente melhorar a trilha de Git, e poder criar um certificado também.

Na trilha tem uma parte:
![image](https://user-images.githubusercontent.com/11020807/218734228-0685edd8-2622-4b2c-bcc9-a9c4c186cbfd.png)

Que a gente queria substituir por exercícios, o que vocês acham que seriam bons exercicios?

**Comment by ArtumosGOC** [comment]
Para conhecimento do Github, parece ser uma boa ideia esse tipo de atividade, ajudaria quem está começando.

**Reply by endersonmenezes** [reply]
@ArtumosGOC mas você acha que necessita de algum exercício a mais, ou somente esses?
